packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-console */
```

Comment:
It will display on the `orbit-components` changelog. The scope is only used to add a **bold** context on the changelog.
On the [current changelog](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/CHANGELOG.md), version 15.8.0 has a feature with no scope (the last one).